frontend/src/main/pages/ProfilePage.js

```diff
@@ -1,20 +1,62 @@
-import React from "react";
-import { Row, Col } from "react-bootstrap";
+import React, { useState, useEffect } from "react";
+import { Row, Col, Modal, Button, Form } from "react-bootstrap";
 import RoleBadge from "main/components/Profile/RoleBadge";
 import { useCurrentUser } from "main/utils/currentUser";
 import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+import {  useBackendMutation } from "main/utils/useBackend";
 
 const ProfilePage = () => {
-
     const { data: currentUser } = useCurrentUser();
+    // Stryker disable all
+    const [showModal, setShowModal] = useState(false);
+    const [phone, setPhone] = useState(() => currentUser?.root?.user?.cellphone);
+    const [newPhone, setNewPhone] = useState(() => currentUser?.root?.user?.cellphone);
+    useEffect(() => {
+        setPhone(currentUser?.root?.user?.cellphone);
+        setNewPhone(currentUser?.root?.user?.cellphone);
+    }, [currentUser]);        
+    // Stryker restore all
+
+    // Stryker disable next-line all
+    const objectToAxiosPutParams = () => ({
+        // Stryker disable all
+        url: "/api/userprofile/updatecellphone",
+        method: "PUT",
+        params: {
+            cellphone: newPhone
+        }
+        // Stryker restore all
+      });
+
+      const onSuccess = (item) => {
+        console.log('User Updated:', item);
+        window.location.reload();
```

Comment:
Forcing a full page reload is bad for UX, but this can be done in a separate PR or for future quarters. At the very least add a todo.
```suggestion
        // TODO: refresh currentUser cache instead of forcing a full page reload
        window.location.reload();
```